openedx/core/djangoapps/theming/management/commands/compile_sass.py

```diff
@@ -92,7 +92,7 @@ def parse_arguments(*args, **options):  # pylint: disable=unused-argument
         if theme_dirs:
             available_themes = {}
             for theme_dir in theme_dirs:
-                available_themes.update({t.theme_dir_name: t for t in get_themes(theme_dir)})
+                available_themes.update({t.theme_dir_name: t for t in get_themes([theme_dir])})
```

Comment:
I changed `get_themes()` to only accept lists as a parameter - this was the only place that needed to be changed.